src/puppet-web/puppet-web.ts

```diff
@@ -455,17 +468,22 @@ export class PuppetWeb extends Puppet {
       destinationId = to.id
     }
 
-    const mediaId = await this.uploadMedia(message, destinationId)
+    const { mediaId, mediaData } = await this.uploadMedia(message, destinationId)
     const msgType = UtilLib.msgType(message.ext())
 
+    mediaData.FileType = 7
```

Comment:
Reference source: https://res.wx.qq.com/a/wx_fed/webwx/res/static/js/index_4a48aef.js

But the use is unknown, and I'll test it again

```
                                var d,
                                    u = {
                                        FromUserName: c.getUserName(),
                                        ToUserName: e.ToUserName,
                                        FileSize: e.size,
                                        FileMd5: t,
                                        FileName: e.name,
                                        FileType: 7
                                    };
                                if (i) {
                                    var m = angular.extend(u, c.getBaseRequest());
                                    d = angular.extend({}, u),
                                        n({
                                            method: "POST",
                                            url: r.API_checkupload,
                                            data: m
                                        }).success(function(t) {
                                            0 == t.BaseResponse.Ret ? (d = angular.extend(d, {
                                                AESKey: t.AESKey,
                                                Signature: t.Signature
                                            }), e.Signature = t.Signature, a(e, t.MediaId, t)) : (e.MMSendMsg && (e.MMSendMsg.MMFileStatus = r.MM_SEND_FILE_STATUS_FAIL, e.MMSendMsg.MMStatus = r.MSG_SEND_STATUS_FAIL), alert(t.BaseResponse.ErrMsg))
                                        }).error(function(t) {
                                            e.MMSendMsg && (e.MMSendMsg.MMFileStatus = r.MM_SEND_FILE_STATUS_FAIL, e.MMSendMsg.MMStatus = r.MSG_SEND_STATUS_FAIL),
                                                alert("上传失败")
                                        })
```